samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -186,13 +209,21 @@ The default value of :kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_SRV_LVL_PROLONG`
 .. note::
    The configuration of light levels, fade time, and timeouts can be changed by altering the configuration parameters in the :file:`prj.conf` file, and rebuilding the sample.
 
+To use the sample with the :guilabel:`Mesh Shell Client` sample, configure the Scheduler, Scene, Time and Light LC Server models of the :guilabel:`Mesh Light Fixture` node as follows:
```

Comment:
Here, I think we might need to bind the key to some of the setup servers as well.